ups/obs_subaru.table

```diff
@@ -7,7 +7,7 @@ setupRequired(pyfits)
 setupRequired(sconsUtils)
 setupRequired(pybind11)
 
-setupOptional(meas_modelfit)
+#setupOptional(meas_modelfit)
```

Comment:
Please add a short comment explaining why this is commented out and what will be required to restore it (both of which presumably amount to "not yet wrapped with pybind11")